src/main/java/Deadlines.java

```diff
@@ -0,0 +1,18 @@
+public class Deadlines extends Task{
+
+    protected String by;
+
+    public Deadlines(String description, String by){
+        super(description);
+        this.by = by;
+    }
+
+    public String getBy(){
+        return by;
+    }
+
+    @Override
+    public String toString(){
+        return ("[D]" + super.toString() + " (by: "+by+")");
```

Comment:
maybe you should explore the use of constants to avoid the need of magic strings.